### ISSUE TITLE
Extra objects during discovery

### ISSUE THREAD

**Issue by rashuge** [brbeaird/SmartThings-MyQ-Edge#22]
I tore down my installation and reconfigured using a container (as a side note, the Github and Docker documentation on configuration and setup are different, was able to get it going though with the Github instructions).  When I did the discovery, it found two MyQ controllers and a Sense controller all of which only showed up when I ran the Docker image.  I removed the Sense controller and one of the MyQ controllers and then configured the remaining MyQ controller and it worked great.  Just giving you a heads up in case others have seen this issue as well.  This is a great project and I appreciate the hard work.

**Comment by brbeaird** [comment]
Yes, it's important to only select the MyQ-Connector driver. Otherwise, you'll get multiple controller devices created, including the Sense controller, which is for a separate integration.

That's a good callout on the Docker instructions, though. I'll update those when I have a chance.
